src/languages/en.js

```diff
@@ -360,6 +361,9 @@ export default {
         payerOwesAmount: ({payer, amount}) => `${payer} owes ${amount}`,
         payerPaidAmount: ({payer, amount}) => `${payer} paid ${amount}`,
         payerSettled: ({amount}) => `paid ${amount}`,
+        payerSettledUp: ({amount}) => `settled up ${amount}`,
```

Comment:
This one should be `paid`, we have changed it since the the design doc, so you can use the copy above.